### DISCUSSION TITLE
How to select all entities at once in an action?

### DISCUSSION THREAD

**Discussion by electricJP** [zachowj/node-red-contrib-home-assistant-websocket#1528] [Q&A] [answered]
Hello all,
Since the latest HA release some of my automations stopped working because the "all" entity does work anymore. How can define an action (light.turn_off) to apply to all entities ? I tried "all", "*" or nothing but it does not work.

Thank you by advance.

**Comment by zachowj** [comment, marked as answer]
I tested "all" as an `entity_id` on version 2024.9.0 and it turned off all lights that were on.

**Reply by electricJP** [reply]
I get an error
![image](https://github.com/user-attachments/assets/5deb27d9-74cb-4a2c-b5e9-5f330f05e06c)

**Comment by zachowj** [comment]
I initially thought you meant the functionality had stopped working. The issue was actually a bug in the issues component, where it incorrectly identified 'all' as an invalid entry.

Was fixed here <https://github.com/zachowj/node-red-contrib-home-assistant-websocket/releases/tag/v0.68.10>

**Comment by electricJP** [comment]
thank you
